hatchet/util/perf_measure.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2017-2023 Lawrence Livermore National Security, LLC and other
+# Hatchet Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+
+import os
+
+try:
+    from pycaliper import annotate_function
+    from pycaliper.instrumentation import begin_region, end_region
+
+    _PYCALIPER_AVAILABLE = True
+except Exception:
+    _PYCALIPER_AVAILABLE = False
+
+try:
+    import perfflowaspect.aspect
+
+    _PERFFLOWASPECT_AVAILABLE = True
+except Exception:
+    _PERFFLOWASPECT_AVAILABLE = False
+
+
+def _validate_perf_plugin(plugin_name):
+    print("Perf Plugin =", plugin_name)
+    print("Caliper is available?", _PYCALIPER_AVAILABLE)
```

Comment:
This print line could be moved inside the if-statement below, and a similar one could be used for perfflowaspect within it's if-statement to be more verbose (see below comment)